auto_update_tests.py

```diff
@@ -112,7 +112,7 @@ def update_wasm_opt_tests():
       actual = ''
       for module, asserts in split_wast(t):
         assert len(asserts) == 0
-        with open('split.wast', 'w') as o:
+        with open('split.wast', 'wb' if t.endswith('.wasm') else 'w') as o:
```

Comment:
Are we writing a binary file into `split.wast` if `t` is a `wasm` file? This feels sketchy...

Reply:
yeah, not sure why the tests do this... probably should be cleaned up.